src/components/Forum/ForumApprovedPostDetails/ForumPostDetailsList.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import {
+  Table, TableContainer, TableBody, TableRow, Typography, withStyles,
+  IconButton,
+} from '@material-ui/core';
+import { withTheme } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import MuiTableCell from '@material-ui/core/TableCell';
+import DeleteIcon from '@material-ui/icons/Delete';
+import EditIcon from '@material-ui/icons/Edit';
+import ReplyIcon from '@material-ui/icons/Reply';
+import CircularProgress from '@material-ui/core/CircularProgress';
+
+const labelsColumnWidth = 160;
+
+const dateFormatOptions = {
+  dateStyle: 'long', timeStyle: 'long',
+};
+
+const TableCell = withStyles({
+  root: {
+    borderBottom: 'none',
+  },
+})(MuiTableCell);
+
+const ActionButtons = withTheme(({ theme }) => (
+  [
+    {
+      Icon: ReplyIcon,
+      label: 'Reply',
+    },
+    {
+      Icon: EditIcon,
+      color: theme.palette.text.main,
+      label: 'Edit',
+    },
+    {
+      Icon: DeleteIcon,
+      color: theme.palette.error.main,
+      label: 'Delete',
+    },
+  ].map(({ Icon, color, label }) => (
```

Comment:
Again, we use the same icons in the approved post list & the approved post details, it'd be great if we could pass data like this action buttons array from the post list to the post details.

Reply:
Good point, this array can just live in a separate file and be imported by whoever needs it.